src/data/polynomial/reverse.lean

```diff
@@ -269,6 +306,19 @@ begin
     exact nat.succ_le_iff.2 (pos_iff_ne_zero.2 hf) }
 end
 
+section eval₂
+variables {S : Type*} [comm_semiring S]
+
+lemma eval₂_reverse_mul_pow (i : R →+* S) (x : S) [invertible x] (f : polynomial R) :
+  eval₂ i (⅟x) (reverse f) * x ^ f.nat_degree = eval₂ i x f :=
+eval₂_reflect_mul_pow i _ _ f (le_refl _)
+
+lemma eval₂_reverse_eq_zero_iff (i : R →+* S) (x : S) [invertible x] (f : polynomial R) :
+  eval₂ i x (reverse f) = 0 ↔ eval₂ i (⅟x) f = 0 :=
+eval₂_reflect_eq_zero_iff i x _ _ (le_refl _)
```

Comment:
Is it worth having the version that swaps `x` and `⅟x`, for convenience?
That might even be a simp lemma.

Reply:
I added this version as a `simp` lemma, and just got rid of the old version.